src/WebJobs.Script/Description/DotNet/FSharp/FSharpCompilationService.cs

```diff
@@ -196,13 +172,17 @@ public ICompilation GetFunctionCompilation(FunctionMetadata functionMetadata)
                     var assembly = Assembly.Load(assemblyBytes, pdbBytes);
                     assemblyOption = FSharpOption<Assembly>.Some(assembly);
                 }
+                else
+                {
+                    _traceWriter.Verbose($"F# Compilation failed with arguments: { string.Join(" ", otherFlags) }");
```

Comment:
In case of failures, we now log the arguments, which will help us identify issues like the duplicate references we ran into.